README.md

```diff
@@ -28,24 +28,29 @@ var counter = postcss.plugin('counter', function () {
 });
 
 var css = 'h1 { foo: 1 }';
-var out = postcss([ filter(), counter(), counter() ]).process(css).css;
+var out = postcss([
+  filter(),
+  counter(),
+  counter()
+]).process(css).css;
 
 console.log(out);
 // => h1 { foo: 2 }
+// note that you will get a PostCSS warning in the message registry
 ```
 
 ## API
 
-### filterPlugins([options])
+### filterPlugins({options})
```

Comment:
The square brackets are to indicate that options are *optional*, can we keep those?